apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -92,6 +94,8 @@ const CurriculumCatalogCard = ({
     onQuickViewClick={onQuickViewClick}
     isInUS={isInUS}
     availableResources={availableResources}
+    isSignedOut={isSignedOut}
```

Comment:
On line 85, should the `?viewAs=Instructor` appended to the course url be conditional on if the user is a student too?